### PR TITLE
Add Pedro Romano to maintainers.

### PR DESCRIPTION
**Pull request by paparomeo**
:wave:

### CODE REVIEW

File: config/maintainers.json [hunk 1]
```diff
@@ -70,6 +70,16 @@
...
       "avatar_url": null,
       "bio": "I'm a Canadian (based in Chicago) golang, C and python developer working on projects within the gaming, finance and data-science worlds respectively. My favorite freetime task is coding challenges and I'm more than happy to help people talk through solutions!"
+    },
+    {
+      "github_username": "paparomeo",
+      "alumnus": false,
+      "show_on_website": false,
+      "name": "Pedro Romano",
+      "link_text": "My (inactive) blog",
+      "line_url": "https://paparomeo.dev/",
```
Comment (cmccandless) [inline config/maintainers.json]:
```suggestion
      "link_url": "https://paparomeo.dev/",
```

Comment (paparomeo) [reply config/maintainers.json]: Oops, silly me! :disappointed: Rebased and fixed and should be good to go now @cmccandless

**Comment by paparomeo** [comment]
Rebased and fixed @cmccandless, Apologies for not checking if the build was passing.

**Comment by cmccandless** [comment]
Merged; Welcome!

For future reference, suggested changes in PRs can be committed directly in GitHub, and commits can be squashed at merge time, so you don't usually need to force-push your branch.

**Comment by paparomeo** [comment]
Thank for merging!

> For future reference, suggested changes in PRs can be committed directly in GitHub, and commits can be squashed at merge time, so you don't usually need to force-push your branch.

Yes, I'm aware, I'm just a bit pedantic about my git history. Thanks for pointing it out, though.

**Pull request merged**